src/logging.ts

```diff
@@ -33,12 +36,18 @@ export function setupLogging( {
 		max_logs: '10',
 		audit_file: path.join( logDir, 'log-rotator.json' ),
 		extension: '.log',
-		create_symlink: true,
+		create_symlink: ! isWindows, // Only create symlinks on non-Windows platforms
 		audit_hash_type: 'sha256',
 		verbose: true, // file-stream-rotator itself will log to console too
 	} );
 
-	currentLogFile = path.join( logDir, 'current.log' );
+	// On Windows, we'll use the most recent log file as the current log file
+	if ( isWindows ) {
+		const today = new Date().toISOString().split( 'T' )[ 0 ].replace( /-/g, '' );
+		currentLogFile = path.join( logDir, `studio-${ today }.0.log` );
```

Comment:
I know our current log size is 5MB, and the chances of reaching that limit are very low, but in theory, it's possible, and in those cases, the '.0.log' will fail. Building the date from scratch seems a bit flaky. What do you think about setting the current log file dynamically?